tests/components/plugwise/test_sensor.py

```diff
@@ -28,7 +28,7 @@ async def test_adam_climate_sensor_entities(hass, mock_smile_adam):
     )
 
     state = hass.states.get("sensor.zone_lisa_wk_battery")
-    assert int(state.state) == 34
+    assert float(state.state) == 0.34
```

Comment:
What kind of measurement is this? Battery sensor measurements should be reported in percent.

Reply:
Fixtures were updated